frontend/packages/console-shared/src/components/dashboard/activity-card/EventItem.tsx

```diff
@@ -40,7 +40,11 @@ const EventItem: React.FC<EventItemProps> = React.memo(({ event, isExpanded, onT
             </div>
             <div className="co-recent-item__title-message">
               {isWarning && (
-                <YellowExclamationTriangleIcon className="co-dashboard-icon co-recent-item__icon--warning" />
+                <YellowExclamationTriangleIcon
+                  alt="Warning alert"
+                  title="Warning alert"
```

Comment:
These are events, not alerts. But I feel like the noun is unnecessarily verbose. Can this just be "Warning"? @jessiehuff ?

```suggestion
                  alt="Warning"
                  title="Warning"
```

If @jessiehuff disagrees, we'd need to change "alert" to "event."